index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio Website</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css" 
+    integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g==" 
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Roboto"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Poppins"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Inter"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=Crete"
+   />
+    <link
+     rel="stylesheet"
+     href="https://fonts.googleapis.com/css?family=R"
+   />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- first  section -->
+    <div class="wrapper">
+      <!-- navbar  section -->
+      <nav class="navbar-section">
+          <span class="nav-span">Victor David</span>
+              <span><a href="#"><img src="./Images/Union.png" alt=""></a></span>
+      </nav>
+      <!-- header  section -->
+      <header>
```

Comment:
- [ ] Great work creating the first section! However, it would be better to use a `section` tag instead of `header` this is a better practice for semantics.